### DISCUSSION TITLE
Unable to open notifications of new release

### DISCUSSION THREAD

**Discussion by HonzaMac** [community/community#29890] [Mobile] [answered]
Anytime I try to read release notification I receive 502. 

It is now happening always for repository renovate/renovate. 

![image](https://user-images.githubusercontent.com/3055648/185295940-b283d626-aec8-4f47-b37d-915e70ddf348.png)

![image](https://user-images.githubusercontent.com/3055648/185295955-4ce8485d-71ab-49b0-983c-89b890821dc6.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone SE 2 (iOS 15.6)
</details>

**Comment by taki-on** [comment]
Thank you for the report. I am wondering if this happens constantly or sometimes. If it happens sometimes, would you mind telling us how often you see the error? This error could happen when you experience a down-graded server performance. Even though we give our best to keep our servers running well, sometimes users experience a down-graded performance. But it is usually fixed pretty soon, so, I am wondering if you saw this error while the server had some performance issues. Sorry for the inconvenience and thank you for the report!

**Reply by HonzaMac** [reply]
When I reported same issue one year ago, it happened just in 80% of cases.
Now it happen always. 
I am able to open changelog/notifications on desktop, but from mobile app it **always** fail. 

Root cause for me seems to be, that mentioned repo has super big release history comparing to other repositories.

**Comment by taki-on** [comment, marked as answer]
Thank you for the further input! I am really sorry for the inconvenience. I could reproduce the issue on our end. I will file this as an API bug and will triage it with the team! Thank you again!

**Reply by HonzaMac** [reply]
Thank you. 

You can link it with my 3 previous bug reports https://support.github.com/ticket/personal/0/1496198

Hope this time it will move forward I could read release notes without problem. 

A lot of libraries are moving to semantic releases with auto publishing of new release for every single commit. Watcher of those repositories will have same problems as me.

**Comment by HonzaMac** [comment]
And one more bug on same release notes:

![image](https://user-images.githubusercontent.com/3055648/192193524-0c4bf555-8740-4a98-aaa0-463dd606fa3e.png)

**Comment by HonzaMac** [comment]
New error message on same release notes

![image](https://user-images.githubusercontent.com/3055648/193450314-c4af9192-5954-4122-b1b2-b1378e020bee.png)

![image](https://user-images.githubusercontent.com/3055648/193450319-182273a4-201c-415d-9614-bad731df002c.png)

![image](https://user-images.githubusercontent.com/3055648/193450323-911a1261-9df3-4d54-bfaa-fe8dc8518bc8.png)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Im getting the same error shown on picture 1 and 3

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@taki-on 

![Screenshot_20221006-082303.png](https://user-images.githubusercontent.com/73130443/194229501-32a65441-6017-4932-ae61-93268baca462.png)

**Comment by HonzaMac** [comment]
Seems problem is fixed! Thank you! ❤️❤️❤️
